### PR TITLE
add external read doc

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by YangKeao — APPROVED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -903,3 +904,53 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+
+下面将详细描述如何在下游 TiDB 集群中读取与上游一致的历史数据。
+
+1. 新建同步任务使用的配置文件，在本示例中该配置文件名为 `test.toml`：
+
+    ```toml
+    # 开启 SyncPoint
+    enable-sync-point = true
+
+    # 每隔 5 分钟写一次 ts-map，这样当下游使用 tidb_enable_external_ts_read 变量读取历史数据的时候
```
Comment (nongfushanquan) [inline ticdc/manage-ticdc.md:922]: 每间隔 5 分钟设置一个全局（所有同步的表）同步的时间点，保证在该时间点时，所有表的数据处于一致性状态。

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -903,3 +904,53 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
 - `tmp-dir` ：指定用于下载 TiCDC 增量数据备份文件的临时目录。
 - `storage` ：指定存储 TiCDC 增量数据备份文件的地址，为 S3 或者 NFS 目录。
 - `sink-uri` ：恢复数据到的下游地址。scheme 仅支持 `mysql`。
+
+## 在下游 TiDB 集群中读取与上游一致的历史数据 <span class="version-mark">从 v6.4.0 版本开始引入</span>
```
Comment (IANTHEREAL) [inline ticdc/manage-ticdc.md:908]:
```suggestion
## 配置只读的灾备集群 <span class="version-mark">从 v6.4.0 版本开始引入</span>
```

Comment (IANTHEREAL) [inline ticdc/manage-ticdc.md:908]:
```suggestion
## 配置只读的灾备集群 <span class="version-mark">从 v6.4.0 版本开始引入</span>
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -903,3 +904,53 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+
+在某些应用场景下，可能需要在数据同步的过程中在下游集群读取与上游某个时刻一致的数据。
```
Comment (IANTHEREAL) [inline ticdc/manage-ticdc.md:910]:
```suggestion
在某些应用场景下，用户业务需要访问 TiCDC 下游的 TiDB 复制集群的数据。 在保证TiDB 复制集群只读的情况，使用 TiCDC  Syncpoint 功能和 TiDB v6.4.0 引入的系统变量 [`tidb_external_ts`](/system-variables.md#tidb_external_ts-从-v640-版本开始引入)，只需通过在下游设置系统变量 [`tidb_enable_external_ts_read`](/system-variables.md#tidb_enable_external_ts_read-从-v640-版本开始引入) 即可方便地读取与上游一致的历史数据。

```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -903,3 +904,53 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+
+TiCDC 提供的 Syncpoint 功能在下游的 TiDB 集群中维护了一个上下游具有一致性快照的 `ts-map`，利用该功能即可够满足这一需求。同时，结合 TiDB v6.4.0 引入的系统变量 [`tidb_external_ts`](/system-variables.md#tidb_external_ts-从-v640-版本开始引入)，你无需手动读取 `ts-map` 的值来使用快照读，而只需通过在下游设置系统变量 [`tidb_enable_external_ts_read`](/system-variables.md#tidb_enable_external_ts_read-从-v640-版本开始引入) 即可方便地读取与上游一致的历史数据。
```
Comment (IANTHEREAL) [inline ticdc/manage-ticdc.md:912]:
```suggestion
```

File: sync-diff-inspector/upstream-downstream-diff.md [hunk 5]
```diff
@@ -75,4 +75,5 @@ select * from tidb_cdc.syncpoint_v1;
 
 - TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。完整的配置请参考 [TiCDC 同步任务配置文件描述](/ticdc/manage-ticdc.md#同步任务配置文件描述)。
 - 需要调整 TiKV 的 GC 时间，保证在校验时 snapshot 对应的历史数据不会被执行 GC。建议调整为 1 个小时，在校验后再还原 GC 设置。
-- 以上配置只展示 `Datasource config` 部分，并不完全。完整配置请参考 [sync-diff-inspector 用户文档](/sync-diff-inspector/sync-diff-inspector-overview.md)。
\ No newline at end of file
+- 以上配置只展示 `Datasource config` 部分，并不完全。完整配置请参考 [sync-diff-inspector 用户文档](/sync-diff-inspector/sync-diff-inspector-overview.md)。
+- TiCDC 从 v6.4.0 开始，使用 Syncpoint 功能需要同步任务拥有下游集群的 SYSTEM_VARIABLES_ADMIN 或者 SUPER 权限。
```
Comment (Oreoxmt) [inline sync-diff-inspector/upstream-downstream-diff.md:79]:
```suggestion
- TiCDC 从 v6.4.0 开始，使用 Syncpoint 功能需要同步任务拥有下游集群的 `SYSTEM_VARIABLES_ADMIN` 或者 `SUPER` 权限。
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -711,6 +712,7 @@ ignore-update-new-value-expr = "gender = 'male' and age > 18" # 过滤掉新值
 * TiCDC v4.0.0 中移除了 `ignore-txn-commit-ts`，添加了 `ignore-txn-start-ts`，使用 start_ts 过滤事务。
 * TiCDC v4.0.2 中移除了 `db-dbs`/`db-tables`/`ignore-dbs`/`ignore-tables`，添加了 `rules`，使用新版的数据库和数据表过滤规则，详细语法参考[表库过滤](/table-filter.md)。
 * TiCDC v6.1.0 及之后移除了 `mounter` 配置项，用户配置该项不会报错，也不会生效。
+* TiCDC 从 v6.4.0 开始，使用 Syncpoint 功能需要同步任务拥有下游集群的 SYSTEM_VARIABLES_ADMIN 或者 SUPER 权限。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:715]:
```suggestion
* TiCDC 从 v6.4.0 开始，使用 Syncpoint 功能需要同步任务拥有下游集群的 `SYSTEM_VARIABLES_ADMIN` 或者 `SUPER` 权限。
```

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -903,3 +905,52 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
 - `tmp-dir` ：指定用于下载 TiCDC 增量数据备份文件的临时目录。
 - `storage` ：指定存储 TiCDC 增量数据备份文件的地址，为 S3 或者 NFS 目录。
 - `sink-uri` ：恢复数据到的下游地址。scheme 仅支持 `mysql`。
+
+## 配置只读的复制集群 <span class="version-mark">从 v6.4.0 版本开始引入</span>
+
+在某些应用场景下，可能需要在数据同步的过程中在下游集群读取与上游某个时刻一致的数据。
+
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:912]:
```suggestion

```

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -903,3 +905,52 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+下面将详细描述如何在下游 TiDB 集群中读取与上游一致的历史数据。
+
+1. 新建同步任务使用的配置文件，在本示例中该配置文件名为 `test.toml`：
+
+    ```toml
+    # 开启 SyncPoint
+    enable-sync-point = true
+
+    # 每间隔 5 分钟设置一个全局（所有同步的表）同步的时间点，保证在该时间点时，所有表的数据处于一致性状态。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:923]:
```suggestion
    # 每间隔 5 分钟设置一个全局（所有同步的表）同步的时间点，保证在该时间点时，所有表的数据处于一致性状态
```

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -903,3 +905,52 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+
+    # 每隔 1 小时清理一次下游 tidb_cdc.syncpoint_v1 表中的 ts-map 数据
+    sync-point-retention = "1h"
+    ```
+
+2. 创建同步任务：
+
+    > **注意：**
+    >
+    > - 由于在读取与上游一致的历史数据时，TiCDC 需要更新下游集群的 `tidb_external_ts` 系统变量，因此请确保创建 `changefeed` 时设置的 `--sink-uri` 中的用户拥有 `SUPER` 或 `SYSTEM_VARIABLES_ADMIN` 权限。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:934]:
```suggestion
    > - 由于在读取与上游一致的历史数据时，TiCDC 需要更新下游集群的 `tidb_external_ts` 系统变量，因此请确保创建 `changefeed` 时设置的 `--sink-uri` 中的用户拥有 `SYSTEM_VARIABLES_ADMIN` 或 `SUPER` 权限。
```

File: ticdc/manage-ticdc.md [hunk 10]
```diff
@@ -903,3 +905,50 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
 - `tmp-dir` ：指定用于下载 TiCDC 增量数据备份文件的临时目录。
 - `storage` ：指定存储 TiCDC 增量数据备份文件的地址，为 S3 或者 NFS 目录。
 - `sink-uri` ：恢复数据到的下游地址。scheme 仅支持 `mysql`。
+
+## 配置只读的复制集群 <span class="version-mark">从 v6.4.0 版本开始引入</span>
+在某些应用场景下，可能需要访问 TiCDC 下游的 TiDB 复制集群的数据。在保证 TiDB 复制集群只读的情况下，利用 TiCDC 的 Syncpoint 功能和 TiDB v6.4.0 引入的系统变量 [`tidb_external_ts`](/system-variables.md#tidb_external_ts-从-v640-版本开始引入)，只需通过在下游设置系统变量 [`tidb_enable_external_ts_read`](/system-variables.md#tidb_enable_external_ts_read-从-v640-版本开始引入) 即可方便地读取与上游一致的历史数据。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:910]:
```suggestion

在某些应用场景下，可能需要访问 TiCDC 下游的 TiDB 复制集群的数据。在保证 TiDB 复制集群只读的情况下，利用 TiCDC 的 Syncpoint 功能和 TiDB v6.4.0 引入的系统变量 [`tidb_external_ts`](/system-variables.md#tidb_external_ts-从-v640-版本开始引入)，只需通过在下游设置系统变量 [`tidb_enable_external_ts_read`](/system-variables.md#tidb_enable_external_ts_read-从-v640-版本开始引入) 即可方便地读取与上游一致的历史数据。
```

File: ticdc/manage-ticdc.md [hunk 11]
```diff
@@ -903,3 +905,50 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+    sync-point-retention = "1h"
+    ```
+
+2. 创建同步任务：
+
+    > **注意：**
+    >
+    > - 由于在读取与上游一致的历史数据时，TiCDC 需要更新下游集群的 `tidb_external_ts` 系统变量，因此请确保创建 `changefeed` 时设置的 `--sink-uri` 中的用户拥有 `SYSTEM_VARIABLES_ADMIN` 或 `SUPER` 权限。
+    > - 如果你需要在下游 TiDB 集群使用 `tidb_enable_external_ts_read` 来读取和上游一致的数据，那么需要确保只有一个 `changefeed` 启用了 Syncpoint 功能。否则，将有可能造成读取到的数据与上游不一致。因为下游 TiDB 集群中只存在一个 `tidb_external_ts` 系统变量，而每个开启了 Syncpoint 功能的 `changefeed` 都会去更新它。并且，请确保 `tidb_enable_external_ts_read = ON` 设置为 session 级别，否则 `changefeed` 将无法往下游继续同步数据。
+    > - 最后，需要保证没有别的地方会对 `changefeed` 同步的表进行写操作，否则无法确保数据一致。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:933]:
```suggestion
    > **注意：**
    >
    > - 由于在读取与上游一致的历史数据时，TiCDC 需要更新下游集群的 `tidb_external_ts` 系统变量，因此请确保创建 `changefeed` 时设置的 `--sink-uri` 中的用户拥有 `SYSTEM_VARIABLES_ADMIN` 或 `SUPER` 权限。
    > - 如果你需要在下游 TiDB 集群使用 `tidb_enable_external_ts_read` 来读取和上游一致的数据，那么需要确保只有一个 `changefeed` 启用了 Syncpoint 功能。否则，将有可能造成读取到的数据与上游不一致。因为下游 TiDB 集群中只存在一个 `tidb_external_ts` 系统变量，而每个开启了 Syncpoint 功能的 `changefeed` 都会去更新它。
```

File: ticdc/manage-ticdc.md [hunk 12]
```diff
@@ -903,3 +905,50 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
...
+
+    ```shell
+    cdc cli changefeed create -c="test" --sink-uri="mysql://root@127.0.0.1:52015/?time-zone=" --config="./test.toml"
+    ```
+
+3. 同步任务开始后，在下游 TiDB 集群中设置 `tidb_enable_external_ts_read` 来读取和上游一致的历史数据：
+
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:940]:
```suggestion

> **注意：**
>
> - 请确保 `tidb_enable_external_ts_read = ON` 设置为 SESSION 级别，否则 `changefeed` 无法向下游继续同步数据。
> - 请确保没有其它连接对 `changefeed` 同步的表进行写操作，否则无法保证数据一致。
```

**Comment by Oreoxmt** [comment]
@IANTHEREAL @nongfushanquan PTAL again

**Review by IANTHEREAL — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 13]
```diff
@@ -902,4 +904,4 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
 
 - `tmp-dir` ：指定用于下载 TiCDC 增量数据备份文件的临时目录。
 - `storage` ：指定存储 TiCDC 增量数据备份文件的地址，为 S3 或者 NFS 目录。
-- `sink-uri` ：恢复数据到的下游地址。scheme 仅支持 `mysql`。
+- `sink-uri` ：恢复数据到的下游地址。scheme 仅支持 `mysql`。
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:907]:
```suggestion
- `sink-uri`：数据恢复后所在的下游地址。scheme 仅支持 `mysql`。
```

Comment (shichun-0415) [reply ticdc/manage-ticdc.md:907]: 上面两行冒号前都多了一个空格

File: ticdc/manage-ticdc.md [hunk 14]
```diff
@@ -900,6 +902,6 @@ cdc redo apply --tmp-dir="/tmp/cdc/redo/apply" \
 
 以上命令中：
 
-- `tmp-dir` ：指定用于下载 TiCDC 增量数据备份文件的临时目录。
-- `storage` ：指定存储 TiCDC 增量数据备份文件的地址，为 S3 或者 NFS 目录。
-- `sink-uri` ：恢复数据到的下游地址。scheme 仅支持 `mysql`。
+- `tmp-dir`：指定用于下载 TiCDC 增量数据备份文件的临时目录。
+- `storage`：指定存储 TiCDC 增量数据备份文件的地址，为 S3 或者 NFS 目录。
+- `sink-uri`：恢复数据到的下游地址。scheme 仅支持 `mysql`。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:907]:
```suggestion
- `sink-uri`：数据恢复的目标地址。scheme 仅支持 `mysql`。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**